docs/Development/Docker.md

```diff
@@ -0,0 +1,60 @@
+Using VIC with Docker
+========
+
+[Docker](http://www.docker.com) is an open-source application container engine. Docker containers allow us to distribute a light-weight, virtual machine that includes the required dependencies to run VIC. VIC developers maintain a single [Ubuntu](http://www.ubuntu.com/) docker image with the minimum requirements to run the VIC **classic** and **image** drivers.
+
+# Getting Started with Docker
+Docker images may be run on most computers running Linux, Mac OS X, or Windows operating systems. To run docker images, **Docker Engine** or "Docker" must be installed on the host computer. The Docker [documentation](https://docs.docker.com/) provides specific instructions for installing Docker for each support operating system.
+
+# Getting the VIC Docker Image
+We provide a single stable light-weight Docker image that includes the minimum requirements to run VIC (e.g. GCC, netCDF, MPI). This image is distributed via [Docker Hub](https://hub.docker.com/) and is called `vic_base_image`.
```

Comment:
GCC is lower case I think: `gcc`

`vic_base_image` is a little confusing with `image` mode (different images). It means that coming up with a different name for `vic_image` would be good (see issue #369).